### PR TITLE
Adding per-field-auth & RDS import tool docs

### PR DESCRIPTION
**Pull request by mikeparisstuff**
*Description of changes:*

Adding docs for per field auth support in the @auth directive. See proposals 1 and 4 here https://github.com/aws-amplify/amplify-cli/issues/1043 for more details. I will continue to add more docs but want to get the PR started.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cli/graphql.md [hunk 1]
```diff
@@ -411,28 +411,42 @@ type Subscription {
...
 }
 enum AuthStrategy { owner groups }
+enum ModelOperation { create update delete read }
+
+# The following objects are deprecated. It is encouraged to use ModelOperations.
 enum ModelQuery { get list }
 enum ModelMutation { create update delete }
 ```
 
+> Note: The operations argument was added to replace the 'queries' and 'mutations' arguments. The 'queries' and 'mutations' arguments will continue to work but it is encouraged to move to 'operations'.
```
Comment (yuth) [inline cli/graphql.md]: Can we include `operations` take precedence over `queries` and `mutations` arguments

File: cli/graphql.md [hunk 2]
```diff
@@ -750,6 +763,69 @@ mutation CreateDraft {
 }
 ```
 
+#### Field Level Authorization
+
+The `@auth` directive specifies that a access to a specific field should be restricted
```
Comment (nikhil-dabhade) [inline cli/graphql.md]: The `@auth` directive specifies that a access to a specific  ..... change to --> "specifies that access to a specific".

File: cli/graphql.md [hunk 3]
```diff
@@ -750,6 +763,69 @@ mutation CreateDraft {
...
+to according to its own set of rules. Here are a few situations where this is useful:
```
Comment (nikhil-dabhade) [inline cli/graphql.md]: "to according" --> "according"

Comment (mikeparisstuff) [reply cli/graphql.md]: fixed

File: cli/graphql.md (hunk 2 above)
Comment (mikeparisstuff) [reply cli/graphql.md]: fixed.

File: cli/graphql.md [hunk 4]
```diff
@@ -2358,6 +2439,118 @@ mutation Delete($noteId: ID!) {
 }
 ```
 
+## Automatically Import Existing DataSources
+
+The Amplify CLI currently supports importing serverless Amazon Aurora MySQL 5.6 databases. Follow these instructions to get started:
```
Comment (nikhil-dabhade) [inline cli/graphql.md]: "Follow these instructions to get started:" --> The following instruction show how to create a Amazon Aurora Serverless database, import this database as a GraphQL data source and test it.

Comment (nikhil-dabhade) [inline cli/graphql.md]: Is this the min version we support or is this the only version that we support for now?

Comment (mikeparisstuff) [reply cli/graphql.md]: fixed

Comment (mikeparisstuff) [reply cli/graphql.md]: AppSync currently only supports Aurora serverless MySQL 5.6 clusters with the data api enabled.

**Pull request merged**